service/src/main/java/bio/terra/pipelines/app/controller/PipelinesApiController.java

```diff
@@ -103,7 +103,7 @@ static ApiPipelineQuota pipelineQuotaToApi(PipelineQuota pipelineQuota) {
         .pipelineName(pipelineQuota.getPipelineName().getValue())
         .defaultQuota(pipelineQuota.getDefaultQuota())
         .minQuotaConsumed(pipelineQuota.getMinQuotaConsumed())
-        .quotaUnits(pipelineQuota.getQuotaUnits().toString());
+        .quotaUnits(pipelineQuota.getQuotaUnits().getValue());
```

Comment:
this _technically_ could be considered a breaking change in like the purest of purity tests but I don't think we care that much about it